test/cppsuite/configs/hs_cleanup_default.txt

```diff
@@ -18,7 +18,8 @@ runtime_monitor=
     (
         enabled=true,
         limit=100
-    )
+    ),
+    postrun_statistics=[cache_hs_insert:0:0, cc_pages_removed:0:0]
```

Comment:
As discussed, stats are broken atm. I'm leaving this here to serve as an example of how to configure the `postrun_statistics` option. I've set it to `0:0` to make sure it breaks as soon as we fix the stats so we remember to find out what the value should be.